hs_core/models.py

```diff
@@ -718,26 +729,25 @@ def update(cls, element_id, **kwargs):
         metadata_obj = kwargs['content_object']
         # get matching resource
         resource = BaseResource.objects.filter(object_id=metadata_obj.id).first()
-
+        publisher_CUAHSI = "Consortium of Universities for the Advancement of Hydrologic Science, Inc. (CUAHSI)"
```

Comment:
@horsburgh What about resources that has no content files? Can those resources be published? If they can be published, should the publisher element allowed to be modified?